src/pages/hoonlee/Login/Login.js

```diff
@@ -1,7 +1,94 @@
-import React from 'react';
+import { React, useState } from 'react';
+import './Login.scss';
+import { Link, useNavigate } from 'react-router-dom';
+import LoginForm from './components/LoginForm/LoginForm';
 
 function LoginHoon() {
-  return <h1>hoonLogin</h1>;
+  const navigate = useNavigate();
+  const [inputId, setInputId] = useState('');
+  const [inputPw, setInputPw] = useState('');
```

Comment:
```suggestion
  const [inputId, setInputId] = useState({
    id: '',
    password : '',
  });
```
- 이렇게 하나의 객체로 사용할 수 있습니다.